packages/amplify-graphql-name-mapping-transformer/src/graphql-refers-to-transformer.ts

```diff
@@ -113,13 +108,15 @@ export const shouldBeAppliedToRDSModels = (
 ): void => {
   const modelName = definition.name.value;
   if (!isSqlModel(ctx, modelName)) {
-    throw new Error(`@${directiveName} is only supported on RDS models. ${modelName} is not an RDS model.`);
+    throw new Error(`@${RefersToDirective.name} is only supported on RDS models. ${modelName} is not an RDS model.`);
```

Comment:
While we're in here:
```suggestion
    throw new Error(`@${RefersToDirective.name} is only supported on SQL models. ${modelName} is not a SQL model.`);
```